constitution.md

```diff
@@ -150,6 +150,8 @@ the membership shall consider at a General Meeting whether that person's members
 
 9.4 That person concerned shall be given a full and fair opportunity to explain matters at the General Meeting. The question of removal shall be determined by a vote of sixty percent (60%) majority of the members present at the General Meeting and must be endorsed by the Clubs and Societies committee.
 
+9.5 If 4 or more members of the Management Committee is terminated or removed, a General Meeting must be called.
```

Comment:
I'd recommend including a time frame for this clause, such as:

> 9.5 If 4 or more members of the Management Committee have resigned or been removed from office since the previous General Meeting, a General Meeting must be called.

Also: must be called for what reason? What business should be transacted at that meeting?